opencog/guile/SchemeSmob.cc

```diff
@@ -232,6 +232,9 @@ void SchemeSmob::register_procs(void*)
 	register_proc("cog-ptv?",              1, 0, 0, C(ss_ptv_p));
 	register_proc("cog-ftv?",              1, 0, 0, C(ss_ftv_p));
 	register_proc("cog-tv->alist",         1, 0, 0, C(ss_tv_get_value));
+	register_proc("cog-tv-mean",           1, 0, 0, C(ss_tv_get_mean));
+	register_proc("cog-tv-confidence",     1, 0, 0, C(ss_tv_get_confidence));
+	register_proc("cog-tv-count",          1, 0, 0, C(ss_tv_get_count));
```

Comment:
I wasn't aware of these helpers, but they do not work the same. The ones I've coded calls the overloaded TruthValue::getMean(), TruthValue()::getConfidence() and TruthValue()::getCount(), while your helpers call the TV alist from the scheme representation. Those calls are different! In my case I can apply cog-tv-mean, etc. to any TV type inheriting TruthValue, even if it's missing from its alist. I need this to easily experiment with TV type conversion.